.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "npm"
+
+      - name: Install
+        run: npm ci
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v4
+        with:
+          browser: chrome
+          headed: true
```

Comment:
Remove `headed`, since by default the Cypress Action runs it in headless mode (without an actual UI of the browser).

Reply:
It was resolved